src/mpris.rs

```diff
@@ -462,8 +463,19 @@ impl MprisPlayer {
     }
 }
 
+/// Commands to control the [MprisManager] worker thread.
+pub enum MprisCommand {
+    /// Notify about playback status and metadata updates.
+    PlaybackUpdated,
+    /// Notify about volume updates.
+    VolumeUpdated,
```

Comment:
Nit: These read more like event names than commands, WDYT?

Reply:
I completely agree. I should have worked a bit more on this PR before creating it as I don't feel 100% happy with it to be honest. It feels rushed. I'm going to turn it into a draft and have another look at it after the 25th (unless you want to fix the issue earlier of course). I'm currently a bit busy with my studies 🫤

Reply:
No worries, as always, appreciate your contributions and take your time. Good luck with your studies!